injector/src/main/java/edu/ucr/cs/riple/injector/changes/Change.java

```diff
@@ -56,6 +77,7 @@ public int hashCode() {
     return Objects.hash(location, annotation);
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
That annotation was for the whole class, but since I was modifying the class, I thought maybe it is better to refine this and not have it for the full class.

Reply:
(For _this_ class, the above is true :) )